modules/developers-guide/examples/bash/get-table-based-token.sh

```diff
@@ -1,4 +1,4 @@
-curl -L -X POST 'http://localhost:8081/v1/auth' \
```

Comment:
Are you certain all of the example codes can switch to `https`?

When I try the basic quick start / developer mode startup command, I am not able to use `https` due to an SSL error. I think this makes sense because that basic mode would not fully configure SSL by default.

Reply:
Yeah, that is why I was asking about whether all stargate apis worked with https. I'll change the auth urls back.

Reply:
@polandll any of the examples that follow the quick start command directly are likely going to require `http`